src/dsp_tools/commands/validate_data/validate_data.py

```diff
@@ -92,25 +81,23 @@ def _inform_about_experimental_feature() -> None:
         "This is an experimental feature, it will change and be extended continuously. "
         "The following information of your data is being validated:",
         "Cardinalities",
+        "If the value type used matches the ontology",
     ]
-    warnings.warn(DspToolsUserWarning(LIST_SEPARATOR.join(what_is_validated)))
+    cprint(LIST_SEPARATOR.join(what_is_validated), color="magenta", attrs=["bold"])
 
 
 def _create_graphs(onto_con: OntologyConnection, data_rdf: DataRDF) -> RDFGraphs:
     ontologies = _get_project_ontos(onto_con)
     knora_ttl = onto_con.get_knora_api()
-    kag = Graph()
-    kag.parse(data=knora_ttl, format="ttl")
-    onto_for_construction = deepcopy(ontologies) + kag
-    card_shapes = construct_cardinality_node_shapes(onto_for_construction)
-    content_shapes = construct_content_shapes_graph(onto_for_construction)
+    knora_api = Graph()
+    knora_api.parse(data=knora_ttl, format="ttl")
+    onto_for_construction = deepcopy(ontologies) + knora_api
+    shapes_graph = construct_shapes_graph(onto_for_construction)
     api_shapes = Graph()
     api_shapes.parse("src/dsp_tools/resources/validate_data/api-shapes.ttl")
-    content_shapes += api_shapes
-    card_shapes += api_shapes
-    return RDFGraphs(
-        data=data_rdf.make_graph(), ontos=ontologies, cardinality_shapes=card_shapes, content_shapes=content_shapes
-    )
+    shapes_graph += api_shapes
+    data_onto = data_rdf.make_graph() + ontologies
+    return RDFGraphs(data_onto=data_onto, ontos=ontologies, shapes=shapes_graph)
```

Comment:
*question* Wouldn't it make sense to keep all graphs separated?

I understand that the current code only needs the data with the ontology.
That is probably why the `data_onto` and the `onto` property both contain the `ontologies` graph which introduces redundant information in the different graphs.
IMHO it might be conceptually "cleaner" and potentially less confusing to keep the `data` graph as a separate properties.
Then you can add a `data_onto` method on `RDFGraphs` to return the combined `data + ontologies` on demand and also a `shapes_onto` method which combines the shapes and the ontology which are used together as well.

Reply:
That was my initial intention, it just got a bit messy. I will change it, but I think it might also be worth it to sit together and discuss naming. I changed it back and dealt with the issue when serialising it into a string for the api.